p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -4876,22 +4877,18 @@ match_kind {
 ~ End P4Example
 
 These identifiers correspond to the P4~14~ match kinds with the same
-names. The semantics of these annotations is actually irrelevant for
-describing the behavior of the P4 abstract machine; how they are used
-influences only the control-plane API and the actual implementation of
-the look-up table. From the point of view of the P4 program, a look-up
-table is an abstract finite map that is given a key and produces as a
-result either an action or a "miss" indication, as described in
-Section [#sec-mau-semantics].
```

Comment:
I'm confused. 

Consider these two tables:
```
table t1 {
  key = { ipv4.dstAddr : ternary }
  actions = { fwd; drop; }
  const entries = { 
     prefix : fwd;
  } 
}
```
and 
table t1 {
  key = { ipv4.dstAddr : exact }
  actions = { fwd; drop; }
  const entries = { 
     prefix : fwd;
  } 
}
```
And suppose that prefix1 encodes `10/8` and prefix2 encodes `192.168/16`.

Applying `t1` will hit on all addresses  in `10/8`, while applying `t2` will hit only for the single address `10.0.0.0`.

Isn't that a difference in data-plane semantics?